apps/web-client/src/routes/orders/customers/+page.svelte

```diff
@@ -0,0 +1,157 @@
+<script lang="ts">
+	import { Plus } from "lucide-svelte";
+	import { createDialog } from "@melt-ui/svelte";
+	import { defaults } from "sveltekit-superforms";
+	import { zod } from "sveltekit-superforms/adapters";
+	import { goto } from "$lib/utils/navigation";
+
+	import { PageCenterDialog, defaultDialogConfig } from "$lib/components/Melt";
+	import CustomerOrderMetaForm from "$lib/forms/CustomerOrderMetaForm.svelte";
+	import { customerOrderSchema } from "$lib/forms";
+	import { data } from "./data";
+	import { getOrderStatus } from "$lib/utils/order-status";
+	import { orderFilterStatus, type OrderFilterStatus } from "$lib/stores/order-filters";
+	import { base } from "$app/paths";
+
+	import Page from "$lib/components/Page.svelte";
+	import { view } from "@librocco/shared";
+
+	const newOrderDialog = createDialog(defaultDialogConfig);
+	const {
+		states: { open: newOrderDialogOpen }
+	} = newOrderDialog;
+
+	const { customers, customerOrderLines } = data;
+
+	$: ordersWithStatus = customers.map((customer) => {
+		const orders = customerOrderLines.filter((line) => line.customer_id === customer.id);
+		const status = getOrderStatus(orders);
+		return {
+			...customer,
+			status
+		};
+	});
+
+	$: hasCompletedOrders = ordersWithStatus.some((order) => order.status === "completed");
+
+	$: filteredOrders = ordersWithStatus.filter((order) => order.status === $orderFilterStatus);
+
+	function setFilter(status: OrderFilterStatus) {
+		orderFilterStatus.set(status);
+	}
+
+	let initialized = true;
+</script>
+
+<Page view={view("orders/customers/id")} loaded={initialized}>
+	<svelte:fragment slot="topbar" let:iconProps let:inputProps>
+		<header class="navbar mb-4 bg-neutral">
+			<input type="checkbox" value="forest" class="theme-controller toggle" />
+		</header>
+	</svelte:fragment>
+
+	<svelte:fragment slot="main">
+		<main class="h-screen">
```

Comment:
If a parent div sets h-screen the children should generally only set h-auto or h-full, otherwise it can remain unresponsive to the parent when the screen resizes